src/components/camera.js

```diff
@@ -79,6 +79,11 @@ module.exports.Component = registerComponent('camera', {
       // Camera disabled. Set camera to another camera.
       system.disableSpectatorCamera();
     }
+
+    this.savedPose = {
+      position: el.getAttribute('position'),
```

Comment:
I think we have to allocate new objects here. You will be holding a reference to the entity shared objects that will change.

Reply:
I just removed it, it wasn't used.